### ISSUE TITLE
Admin and Managed Servers Not Starting

### ISSUE THREAD

**Issue by bdueck** [oracle/weblogic-kubernetes-operator#120]
The Admin server and Managed Servers are continually failing to start and showing only the following entry (with slight variations in directory and file name) in each the pod's log. I'm running Docker/K8S on MacOS using Docker-edge.

failed to open log file "/var/log/pods/bc42ce99-1bd8-11e8-9d6b-025000000001/weblogic-server_17.log": open /var/log/pods/bc42ce99-1bd8-11e8-9d6b-025000000001/weblogic-server_17.log: no such file or directory

Any advice is appreciated.

**Comment by rjeberhard** [comment]
Can you please tell us about the persistent volume you configured?  Did the domain creation job run successfully?  The directory behind the persistent volume (host path) has to be chmod 777 accessible since the WebLogic pods will run as user = "weblogic", which I believe is uid=1000.

**Comment by bdueck** [comment]
Yes, the domain creation job ran successfully - a visual inspection of what was created shows that things look correct. 

The persistent volume is under /Users/<username>/k8s_dir. Under there I have persistentVolume001 and persistentVolume002. Everything is 777. 

I did previously have a problem where I wasn't creating the volume under /Users and then noticed the note that this was a problem for a MacOS Docker/K8S setup. This was giving an error even creating the domain. Once I moved it under /Users then domain creation worked, but then lead to this error when the servers try to startup.

**Comment by tbarnes-us** [comment]
I've heard that it's fairly common on a mac that the vm memory allocation needs to be increased.

**Comment by doxiao** [comment]
@bdueck, has the issue been resolved?
